plugins/callback/logger_runtime.py

```diff
@@ -0,0 +1,152 @@
+# -*- coding: utf-8 -*-
+# (c) 2020 Ashish Varshney <er.ashish.av@gmail.com>
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+import collections
+import time
+import logging
+from ansible.plugins.callback import CallbackBase
+
+
+DOCUMENTATION = '''
+    callback: logger_runtime
+    type: aggregate
+    short_description: Adds time information to tasks
+    version_added: "2.0"
+    description:
+      - Ansible callback plugin for timing individual tasks and overall execution time.
+    requirements:
+      - Whitelisting in configuration.
```

Comment:
```suggestion
      - enable callback in configuration.
```

Reply:
+1 to this

Reply:
I made the `e` capital, so it's a proper sentence :)